nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
```

Comment:
It sounds like portability is an important quality of code in common_msgs. Given that map_server has two purposes: load a map from a file (it takes a file path as an argument when it is launched), and offer a map information on /map and /map_metadata topics,  are you saying you'd like to see those two purposes split?

I can refactor map_server to only offer the map, and create a separate loader node to give that map to map server. It would then be possible for other loaders to be created in support of other systems.